index.bs

```diff
@@ -1106,7 +1111,7 @@ The <dfn method for=FileSystemSyncAccessHandle>read(|buffer|, {{FileSystemReadWr
 The <dfn method for=FileSystemSyncAccessHandle>write(|buffer|, {{FileSystemReadWriteOptions}}: |options|)</dfn> method steps are:
 
 1. If [=this=].[=[[state]]=] is "`closed`", throw an {{InvalidStateError}}.
-1. Let |writePosition| be |options|.{{FileSystemReadWriteOptions/at}}.
+1. Let |writePosition| be |options|.{{FileSystemReadWriteOptions/at}}, if specified, else the current file offset.
```

Comment:
```suggestion
1. Let |writePosition| be |options|["{{FileSystemReadWriteOptions/at}}"], if given; otherwise [=this=]'s [=FileSystemWritableFileStream/file position cursor=].
```